main.js

```diff
@@ -0,0 +1,132 @@
+//multiply number vs 1 to 10
+
+const multiplyNum = num => {
+    console.log('In bang nhan:')
+    for (let i = 1; i < 11; i++) {
+        console.log(num + ' x ' + i + ' = ' + ' ' + num * i)
+    }
+}
+
+//find even number
+
+const findEvenNum = num => {
+    if (num >=1 && num <=30) {
+        for (let i = 1; i < 31; i++) {
+            if (i % 2 === 0) {
+                console.log(i)
+            }
+            if (i >= num) break
+        }
+    } else {
+        console.log('Bạn cần nhập số từ 1 đến 30')
+    }
+}
+
+//sum number
+
+const sumNum = num => {
+    if (num >=1 && num <=30) {
+        let sum = 0
+        for (let i = 1; i < 31; i++) {
+            sum += i
+            if (i >= num) break
+        }
+        console.log(sum)
+    } else {
+        console.log('Bạn cần nhập số từ 1 đến 30')
+    }
+}
+
+// caculate factorial number
+
+const calculateFactorialNum = num => {
+    if (num >=1 && num <=30) {
+        let factorial = 1
+        for (let i = 1; i < 31; i++) {
+            factorial *= i
+            if (i >= num) break
+        }
+        console.log(factorial)
+    } else {
+        console.log('Bạn cần nhập số từ 1 đến 30')
+    }
+}
+
+// count even number for array
+
+const countEvenNumForArr = Array => {
+    let count = 0
+    Array.forEach(item => {
+        if(item % 2 === 0) {
+            count++
+        }
+    })
+    console.log(count)
+}
+
+// delete duplicate
+
+const deleteDuplicate = arr => {
+    let checkForSurvival = (arr, item) => arr.indexOf(item) > -1
+    let arrNew = []
```

Comment:
```suggestion
    const arrNew = []
```
`arrNew` ko bị gán lại, nên hãy đặt là `const`